photon-client/src/integTest/scala/com/linkedin/photon/ml/DriverTest.scala

```diff
@@ -29,7 +29,7 @@ import com.linkedin.photon.ml.PhotonOptionNames._
 import com.linkedin.photon.ml.TaskType.TaskType
 import com.linkedin.photon.ml.constants.MathConst
 import com.linkedin.photon.ml.diagnostics.DiagnosticMode
-import com.linkedin.photon.ml.io.{FieldNamesType, GLMSuite, InputFormatType}
+import com.linkedin.photon.ml.photon_io.{FieldNamesType, InputFormatType}
```

Comment:
The idea was to isolate "photon" from "game" stuff, this photon_io dir being on its way out when we finalize the transition to game.